src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -91,21 +91,28 @@ class SideNav extends Component {
     }
 
     return (
-      <ul
+      <div
+        id="va-sidenav-wrapper"
```

Comment:
If we aren't using this `id` in JS and only in CSS, can we make this a class name instead?

Reply:
Sure can, @kelsonic